lib/noun-project-api.rb

```diff
@@ -1,10 +1,14 @@
 require 'oauth'
 require 'json'
 require 'noun-project-api/icon_retriever'
+require 'noun-project-api/reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.